src/helper.js

```diff
@@ -0,0 +1,32 @@
+import { v4 as uuidv4 } from 'uuid';
+import render from './dom';
+
+function NewProject(name) {
+  this.id = uuidv4();
+  this.name = name;
+}
```

Comment:
Please use the arrow function here.

Reply:
We added the function keyword because constructor functions can not be arrow functions.